### PR TITLE
KBDFans Tiger80 Keymap

### PR DESCRIPTION
**Pull request by micros24**
## Description

RGB underglow is disabled as I don't use it.
Keymap by Micros24 for optimized gaming performance and media controls.
SCROLL LOCK and PAUSE BREAK keys have been replaced with VOL- and VOL+ respectively.
F13 have been replaced with PLAY/PAUSE MEDIA.
RWIN is configured as F9 + ENTER macro.
RALT and RCTRL have tap and hold configured as follows:
RALT:
- TAP is configured as PREVIOUS MEDIA
- HOLD is configured as normal RALT

RCTRL:
- TAP is configured as NEXT MEDIA
- HOLD is configured as normal RCTRL

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kbdfans/tiger80/keymaps/micros24/config.h [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+#include "config_common.h"
+
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 16
+#define MATRIX_ROW_PINS { B0, E6, B1, B4, D1, D2 }
+#define MATRIX_COL_PINS { F7, F6, F5, F4, F1, F0, D3, D5, D4, D6, D7, B5, B6, C6, E2, D0 }
+
+#define DIODE_DIRECTION COL2ROW
+
```
Comment (waffle87) [inline keyboards/kbdfans/tiger80/keymaps/micros24/config.h:27]:
```suggestion
```

File: keyboards/kbdfans/tiger80/keymaps/micros24/config.h [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	#define RGBLED_NUM 20
+	#define RGBLIGHT_HUE_STEP 10
+	#define RGBLIGHT_SAT_STEP 10
+	#define RGBLIGHT_VAL_STEP 10
+	#define RGBLIGHT_SLEEP
+#endif
+*/
+
+#define VIA_EEPROM_LAYOUT_OPTIONS_SIZE 2
+
```
Comment (waffle87) [inline keyboards/kbdfans/tiger80/keymaps/micros24/config.h:80]:
```suggestion
```

File: keyboards/kbdfans/tiger80/keymaps/micros24/config.h [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+/* Increasing performance*/
+#define MATRIX_IO_DELAY 20
+#define USB_POLLING_INTERVAL_MS 1
```
Comment (waffle87) [inline keyboards/kbdfans/tiger80/keymaps/micros24/config.h:83]:
```suggestion
```

File: keyboards/kbdfans/tiger80/keymaps/micros24/keymap.c [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
```
Comment (waffle87) [inline keyboards/kbdfans/tiger80/keymaps/micros24/keymap.c:16]:
```suggestion
```

File: keyboards/kbdfans/tiger80/keymaps/micros24/readme.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+* **Bootmagic reset**: Hold down the key at (0,0) in the matrix
+* **Physical reset button**: Briefly press the button on the back of the PCB
+* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
```
Comment (waffle87) [inline keyboards/kbdfans/tiger80/keymaps/micros24/readme.md:22]:
```suggestion
```

File: keyboards/kbdfans/tiger80/keymaps/micros24/rules.mk [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+
+DEBOUNCE_TYPE = sym_eager_pk
+VIA_ENABLE = yes
+LT0_ENABLE = yes
+
+LAYOUTS = tkl_f13_ansi_tsangan
```
Comment (waffle87) [inline keyboards/kbdfans/tiger80/keymaps/micros24/rules.mk:24]:
```suggestion
MOUSEKEY_ENABLE = no
RGBLIGHT_ENABLE = no
DEBOUNCE_TYPE = sym_eager_pk
VIA_ENABLE = yes
LTO_ENABLE = yes
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/kbdfans/tiger80/keymaps/micros24/rules.mk (hunk 6 above)
Comment (waffle87) [reply keyboards/kbdfans/tiger80/keymaps/micros24/rules.mk]: > `LT0_ENABLE = yes`

This still needs to be resolved — `LTO_ENABLE`

File: keyboards/kbdfans/tiger80/keymaps/micros24/rules.mk [hunk 7]
```diff
@@ -0,0 +1,5 @@
+RGBLIGHT_ENABLE = no
+MOUSEKEY_ENABLE = no
+DEBOUNCE_TYPE = sym_eager_pk
+VIA_ENABLE = yes
+LT0_ENABLE = yes
```
Comment (waffle87) [inline keyboards/kbdfans/tiger80/keymaps/micros24/rules.mk:5]:
```suggestion
LTO_ENABLE = yes
```
this still needs to be resolved.

**Review by fauxpark — APPROVED** [review]

**Pull request merged**